src/transport/SessionManager.cpp

```diff
@@ -842,16 +801,17 @@ void SessionManager::UnauthenticatedMessageDispatch(const PacketHeader & partial
     // Associate the unauthenticated session with the connection, if not done already.
     if (peerAddress.GetTransportType() == Transport::Type::kTcp)
     {
-        Transport::ActiveTCPConnectionState * sessionConn = unsecuredSession->GetTCPConnection();
-        if (sessionConn == nullptr)
+        Transport::ActiveTCPConnectionHolder sessionConn = unsecuredSession->GetTCPConnection();
+        if (sessionConn.IsNull())
         {
             unsecuredSession->SetTCPConnection(ctxt->conn);
         }
         else
         {
             if (sessionConn != ctxt->conn)
             {
-                ChipLogError(Inet, "Data received over wrong connection %p. Dropping it!", ctxt->conn);
+                ChipLogError(Inet, "Unauthenticated data received over %p for wrong connection %p. Dropping it!",
```

Comment:
This new log line is not making sense to me.  The message was received over ctxt->conn, no?  sessionConn is where we were _expecting_ to get data for this session... and didn't.

Should be something more like `Unauthenticated data received over TCP connection %p instead of %p` and reverse the order of the arguments being passed for those `%p`?

Reply:
Sorry I just tweaked the message so it's not the same as the other (same log text used twice in this file) & based it on the method name, `UnauthenticatedMessageDispatch` - this was just so that during debugging it would be clear where this is being logged 😅

Reply:
My point is that the new text does not make sense.